python/pyspark/mllib/linalg.py

```diff
@@ -440,7 +440,7 @@ def __init__(self, size, *args):
         values (sorted by index).
 
         :param size: Size of the vector.
-        :param args: Non-zero entries, as a dictionary, list of tupes,
+        :param args: Non-zero entries, as a dictionary, list of tuples,
```

Comment:
This is not very accurate because we can take explicit zeros. Could you change it to

`Active entries, as a dictionary {index: value, ...}, a list of tuples [(index, value), ...], or a list of strictly increasing indices and a list of values [index, ...], [value, ...]. Inactive entries are treated as zeros.`

It wold be nice if you can also update the Scala doc (in a separate PR).